### ISSUE TITLE
CSS Cascade Layers

### ISSUE THREAD

**Issue by lilles** [mozilla/standards-positions#471]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Cascade Layers (CSS Cascading and Inheritance Level 5)
* Specification or proposal URL: https://drafts.csswg.org/css-cascade-5/#layering
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): 

### Other information

* https://css.oddbird.net/layers/

**Comment by bholley** [comment]
I'm generally supportive of the idea, but would like to hear from the layout team. Perhaps @emilio would like to comment?

**Comment by emilio** [comment]
Yeah, agreed. Generally the specification is still in early stages, but the goal seems worth pursuing.

**Comment by bholley** [comment]
@emilio Mind making a quick PR to close this out?

**Comment by SebastianZ** [comment]
For what it's worth, I've just created a tracking bug for CSS Cascade 5:
https://bugzilla.mozilla.org/show_bug.cgi?id=1699214

Sebastian
